internal/query/projection/user_grant.go

```diff
@@ -439,9 +442,8 @@ func getResourceOwners(ctx context.Context, es handler.EventStore, instanceID, u
 	if err != nil {
 		return "", "", "", err
 	}
-	if len(events) != eventCount {
```

Comment:
eventCount can be removed and `reduceAddded` (in this file) does not yet check if the necessary resource owners are returned

Reply:
will remove eventCount, but if all necessary resourceOwners are returned is checked in the places where this function is used as they are variable

Reply:
https://github.com/zitadel/zitadel/pull/7383/files#diff-2e359312ea0b5419c077502f30b3ce835862614a1f11a399e1690ce8d60b6204R173-R176

Reply:
If I try to open the link I just get this PRs file changes, what did you want to highlight?

Reply:
totally missed that, added a function with error on missing resourceowner checks.